server/views/pages/appointments/create-and-edit/upload-prisoner-list.njk

```diff
@@ -18,20 +19,31 @@
                     id: "file",
                     name: "file",
                     label: {
-                        text: "Upload a CSV file of prison numbers",
+                        text: "Upload your list of prison numbers",
                         classes: "govuk-label--l",
                         isPageHeading: true
                     },
                     hint: {
-                        text: "Upload a CSV file with the prison numbers of all people attending this appointment"
+                        text: "Your list of prison numbers must be in a CSV file"
                     },
                     errorMessage: validationErrors | findError('file')
                 }) }}
-
+                 {{ govukDetails({
+                                  summaryText: "How to use a CSV file",
+                                  text: howToUseCSVFile()
```

Comment:
While it does seem to work using text, the html property would be more appropriate due to the paragraph tags

Reply:
changed from text to html